swan-lake/learn/keeping-ballerina-up-to-date.md

```diff
@@ -147,30 +133,16 @@ Use 'ballerina help dist <command>' for more information on a specific command.
 
 Most of these subcommands are self-explanatory. Therefore, the following sections introduce them briefly.
 
-### Update to the latest patch version
```

Comment:
This section is still valid as we will be doing 1.2.x releases and it will still update to latest patch.
In swan lake it will bump to next preview